packages/recomponents/src/components/r-tabs/r-tab.vue

```diff
@@ -27,6 +36,17 @@
                 isActive: this.active,
             };
         },
+        computed: {
+            computedAccessibilityId() {
+                return this.panelId || this._uid;
+            },
+            computedPanelId() {
+                return generateTabpanelId(this.computedAccessibilityId);
+            },
+            computedTabId() {
```

Comment:
I don't think the `computed` prefix is needed here, it's not a convention we follow

Reply:
I'd think it's because of line 22, but generally agree it's not a prefix we need 😄

Reply:
Removed it :)